Assets/_Application/Layers/LayerTypes/Polygons/Properties/ScatterGenerationSettingsPropertyData.cs

```diff
@@ -37,12 +39,13 @@ public ScatterGenerationSettingsPropertyData(string originalObjectPrefabId, floa
             this.maxScale = maxScale;
             this.fillType = fillType;
             this.strokeWidth = strokeWidth;
-            AutoRotateToLine = autoRotateToLine;
+            AutoRotateToLine = autoRotateToLine; //todo: remove this
```

Comment:
Is todo nog relevant?

Reply:
ja, dat waren we vergeten, is nu opgelost